.github/workflows/tests.yml

```diff
@@ -1,112 +1,42 @@
-name: Run tests with worflow_call
```

Comment:
What is the plan with the disabled workflows?

Reply:
Will be fixed in DXJ-216